doc/_static/mpl.css

```diff
@@ -247,13 +250,16 @@ div.sphinxsidebar ul {
     list-style: none;
     padding: 0;
     line-height: 130%;
+    
 }
 
 div.sphinxsidebar ul ul {
     list-style: square;
     margin-top: 6px;
     margin-bottom: 6px;
-    margin-left: 16px;
+    margin-left: 24px;
```

Comment:
If `margin-left` was left at 16px, would word-wrap won't work?

Reply:
Its breaking the word in hetween like tight_layout is coming as 
tight_la
yout

This doesn't look good hence I had to increase the width